### PR TITLE
Add --shard option to doctrine command

### PR DESCRIPTION
**Pull request by vincentchalamon**
This PR requires https://github.com/doctrine/dbal/pull/896, and is required by https://github.com/doctrine/DoctrineBundle/pull/456

- [x] Add `--shard` option to `doctrine:fixtures` commands

### CODE REVIEW

File: Command/LoadDataFixturesDoctrineCommand.php [hunk 1]
```diff
@@ -74,6 +76,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if ($input->getOption('shard')) {
+            if (!$em->getConnection() instanceof PoolingShardConnection) {
+                throw new \LogicException(sprintf("Connection of EntityManager '%s' must implements shards configuration.", $input->getOption('em')));
```
Comment (guillaumevoisin) [inline Command/LoadDataFixturesDoctrineCommand.php]: "must implement"

**Comment by guillaumevoisin** [comment]
:+1:

**Comment by MLKiiwy** [comment]
:+1:

**Comment by vincentchalamon** [comment]
@iamluc Can you also have a look at it ? Thanks :)

**Comment by vincentchalamon** [comment]
@deeky666 Can you also have a look at it please ? :)

**Comment by dunglas** [comment]
:+1:

**Comment by vincentchalamon** [comment]
Thanks @guilhermeblanco

**Pull request merged**